plugin-build/src/main/kotlin/io/sentry/android/gradle/AppConfig.kt

```diff
@@ -271,6 +333,7 @@ private fun ApplicationVariant.configureProguardMappingsTasks(
 private fun ApplicationVariant.configureNativeSymbolsTask(
```

Comment:
Need to merge with `main` and change this a bit after resolving conflicts, we extracted task configuration into its own `.register` method similar to the others